.circleci/config.yml

```diff
@@ -526,6 +526,29 @@ jobs:
             rm -rf src/main/java src/test/java src/it/java
             mvn -Dakkaserverless-sdk.version=$SDK_VERSION test-compile
 
+
+  sample-java-eventsourced-counter-pojo:
```

Comment:
I duplicated the counter samples because I wanted to have at least something to run integration tests and prove that the serialization is working as expected. 

Needed duplication because domain proto (that is deleted in this sample) is used in the docs.